### ISSUE TITLE
Ruby: <Mastermind project of the OOP section lacks the resources to explain the rules of the game>

### ISSUE THREAD

**Issue by NewDev2005** [TheOdinProject/curriculum#28222]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

I had never heard of the Mastermind game until i started working on the project. In the mastermind project section there is no external resource for learning the rules of the game except the wikipedia page which is not really helpful, so i had to google multiple times and then i found out an article which explained the rules of the game very nicely and in a very simple way. The people who are doing the odin project comes from the different countries some are aware of the game and some are not. so its really essential to get accquainted with the rules of the game and how the game works before we start working on the project. This is the article which i found very helpful and useful  https://www.wikihow.com/Play-Mastermind  this article would really help the students  to clear all their doubts regarding the game. 

### Path

Ruby / Rails

### Lesson Url

https://www.theodinproject.com/lessons/ruby-mastermind

### (Optional) Discord Name

Htmx

**Comment by kahmoh** [comment]
If no one is working on this, I'm happy to. I'd like to suggest this resource as an alternative. [https://en.wikipedia.org/wiki/Mastermind_(board_game)](url). It is more in line with other links for game rules in the curriculum, such as the battleship game in the JS path.

**Comment by Eduardo06sp** [comment]
The Wikihow link explains it much more clearly & concisely than the Wikipedia page does.

I relied on the Wikihow article to actually understand the game. The Wikipedia page is harder to follow along.

**Comment by Eduardo06sp** [comment]
There is already a link to the Wikipedia page in the project page as well.

**Comment by NewDev2005** [comment]
exactly

**Comment by kahmoh** [comment]
Understood.

**Comment by JoshDevHub** [comment]
I think I'd be okay adding the wikihow link in. The game's rules do commonly confuse people in the discord.

I'll assign you and you can add the link @kahmoh :rocket:
